### PR TITLE
py3 update

### PR DESCRIPTION
**Pull request by jburel**
first round for py3

### CODE REVIEW

**Comment by joshmoore** [comment]
Only 3 integration tests left, possibly related:

```
>               assert o['Message'] == 'Script Completed'
E               KeyError: 'Message'
```

https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/198/#showFailuresLink

**Comment by jburel** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/222/#showFailuresLink No scripts related failure
Same on py3

**Comment by joshmoore** [comment]
Only script failures in py3 were coming from client code. :+1:

File: omero/export_scripts/Batch_ROI_Export.py [hunk 1]
```diff
@@ -305,9 +305,9 @@ def run_script():
             description="List of Dataset IDs or Image IDs").ofType(rlong(0)),
 
         scripts.List(
-            "Channels", grouping="3", default=[1L, 2L, 3L, 4L],
+            "Channels", grouping="3", default=[1, 2, 3, 4],
             description="Indices of Channels to measure intensity."
-            ).ofType(rlong(0)),
```
Comment (joshmoore) [inline omero/export_scripts/Batch_ROI_Export.py:310]: In general, `int` makes sense here, but it will impact clients, right?

Comment (jburel) [reply omero/export_scripts/Batch_ROI_Export.py:310]: I will have to double check

Comment (will-moore) [reply omero/export_scripts/Batch_ROI_Export.py:310]: I would think that webclient should handle this fine since the UI for that script is not hard-coded.

**Comment by jburel** [comment]
Merging to avoid conflict

**Pull request merged**